contracts/contracts/coordination/Coordinator.sol

```diff
@@ -144,6 +143,10 @@ contract Coordinator is Initializable, AccessControlDefaultAdminRulesUpgradeable
         return rituals[ritualId].accessController;
     }
 
+    function getFeeModel(uint32 ritualId) external view returns (IFeeModel) {
+        return rituals[ritualId].feeModel;
```

Comment:
Something occurred to me. What happens to the existing rituals on `mainnet` that would not have a feeModel set in their `Ritual` struct? I guess those can't be extended? Other side effects?

Reply:
Oh wait. Fee model was on the old Ritual struct. Nevermind.

Reply:
you are correct, old rituals can't be extended, I can create work around with FlatFeeRateModel as default for old rituals, wdyt?

Reply:
Opened #282, so let's move on with the PR.